src/components/PageComponents/Wthr.jsx

```diff
@@ -25,31 +27,31 @@ function Wthr() {
             />
             <div className="flex flex-col xl:gap-1 2xl:gap-2">
               <p className="text-2xl font-bold text-center xl:text-3xl 2xl:text-5xl text-primary-tw dark:text-dark-primary-tw">
-                {data && data.current.feelslike_f}
+                {data && data.current.temp_f}
               </p>
               <p className="text-xs text-center xl:text-sm text-primary-tw dark:text-dark-primary-tw">
                 {data && data.current.condition.text}
               </p>
             </div>
           </div>
           {/* Weather Details Wrapper */}
-          {data && astroData && (
+          {data && forecastData && (
             <div className="grid items-center grid-cols-2 gap-2 w-fit xl:gap-3 2xl:gap-3">
               {/* Weather Key Pair */}
               <WeatherKeyInfo
                 svg="sunrise"
-                text={astroData.astronomy.astro.sunrise}
+                text={forecastData.forecast.forecastday[0].astro.sunrise}
```

Comment:
I would factor out 'forecastData.forecast.forecastday[0]' another variable. This could give some better clarity on what actually represents when someone else is reading the code.